src/Stratis.Bitcoin.Features.Miner/Controllers/MiningController.cs

```diff
@@ -70,8 +59,9 @@ public IActionResult Generate([FromBody]MiningRequest request)
 
             try
             {
-                if (this.network.Consensus.IsProofOfStake)
-                    return ErrorHelpers.BuildErrorResponse(HttpStatusCode.MethodNotAllowed, "Method not allowed", "Method not available for Proof of Work");
+                if (this.network.Consensus.IsProofOfStake &&
+                    this.consensusManager.Tip.Height > this.network.Consensus.LastPOWBlock)
+                    return ErrorHelpers.BuildErrorResponse(HttpStatusCode.MethodNotAllowed, "Method not allowed", string.Format("This is a POS node and it's consensus tip is higher than the allowed LastPowBlock height of {0}", this.network.Consensus.LastPOWBlock));
```

Comment:
Instead of `string.Format("This is a POS node and it's consensus tip is higher than the allowed LastPowBlock height of {0}"`

we can use `$"This is a POS node and it's consensus tip is higher than the allowed LastPowBlock height of {this.network.Consensus.LastPOWBlock}"`

Reply:
Ah yes 👍